scripts/release/monitor/requirements.txt

```diff
@@ -0,0 +1,47 @@
+#
+# This file is autogenerated by pip-compile with Python 3.10
+# by the following command:
+#
+#    pip-compile
+#
+annotated-types==0.5.0
+    # via pydantic
+certifi==2023.5.7
+    # via requests
+charset-normalizer==3.2.0
+    # via requests
+click==8.1.5
+    # via typer
+colorama==0.4.6
+    # via typer
+idna==3.4
+    # via requests
+markdown-it-py==3.0.0
+    # via rich
+mdurl==0.1.2
+    # via markdown-it-py
+pydantic==2.0.3
```

Comment:
WARNING: you are using pydantic 2! See https://github.com/ITISFoundation/osparc-simcore/issues/4481